src/flicking.js

```diff
@@ -775,6 +779,41 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 		},
 
 		/**
+		 * Set container's height according to each children's height
+		 * @param direction
+     */
+		_setAdaptiveHeight: function(direction) {
+			var MC = eg.MovableCoord;

```

Comment:
you can use 'MC' without declare variable because eg.MovableCoord is injected in Flicking.

plz. remove this line.